kernelci/configs.py

```diff
@@ -534,6 +534,12 @@ def __init__(self, name, mach, arch='arm', *args, **kw):
         kw.setdefault('dtb', '{}.dtb'.format(name))
         super(DeviceType_arm, self).__init__(name, mach, arch, *args, **kw)
 
+class DeviceType_mips(DeviceType):
```

Comment:
Please keep 2 blank lines before the beginning of each class, as reported by `pep8`:
```
kernelci/configs.py:537:1: E302 expected 2 blank lines, found 1
kernelci/configs.py:544:1: E302 expected 2 blank lines, found 1
```